nominal/core.py

```diff
@@ -709,6 +804,35 @@ def create_video_from_io(
         response = self._ingest_client.ingest_video(self._auth_header, request)
         return self.get_video(response.video_rid)
 
+    def create_log_set(
+        self,
+        name: str,
+        logs: Iterable[Log] | Iterable[tuple[datetime | IntegralNanosecondsUTC, str]],
+        timestamp_type: LogTimestampType = "absolute",
+        description: str | None = None,
+    ) -> LogSet:
+        """
+        Creates a log set, to which logs can be attached using `attach-and-finalize`. The logs within a logset are
```

Comment:
nit: use imperative mood / command language for docstrings:

> The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.


https://peps.python.org/pep-0257/

Reply:
also this docstring is now inaccurate

Reply:
updating